mixed_requests/mixed_test.py

```diff
@@ -17,6 +17,10 @@ def create_clients(self):
         self.wrk.set_script(self.script)
         self.clients = [self.wrk]
 
+    # temporary until fixed
```

Comment:
Comments should be formatted  as normal sentence: starting with capital letter and ending with punctuation mark.